rest/src/main/java/discord4j/rest/entity/RestGuild.java

```diff
@@ -68,7 +68,9 @@ static RestGuild create(RestClient restClient, long id) {
      * entity. If an error is received, it is emitted through the {@code Mono}.
      */
     public Mono<GuildUpdateData> getData() {
-        return restClient.getGuildService().getGuild(id);
+        Map<String, Object> queryParams = new HashMap<>();
+        queryParams.put("with_counts", false);
+        return restClient.getGuildService().getGuild(id, queryParams);
```

Comment:
Your last commit removed `getData(@Nullable Boolean)` method, please readd it and make this method call `getData(null)`